localization/english/opinions/wc_general_opinions_l_english.yml

```
@@ -1,22 +1,21 @@
 ﻿l_english:
- being_undead_crime:0 "[GetTrait('being_undead').GetName( GetNullCharacter )]"
- being_undead_intolerant:0 "[GetTrait('being_undead').GetName( GetNullCharacter )]"
- 
- loyal_undead:0 "Loyal [GetTrait('being_undead').GetName( GetNullCharacter )]"
+ being_undead_crime:0 "[GetTrait('being_undead').GetName( GetNullCharacter )|V]"
+ being_undead_intolerant:0 "[GetTrait('being_undead').GetName( GetNullCharacter )|V]"
+ loyal_undead:0 "Loyal [GetTrait('being_undead').GetName( GetNullCharacter )|V]"
```

One empty line should be here. Sorry if it looks too minor to you, but localization rules are pretty strict to avoid conflicts in the future.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/Localization-Guidelines
```suggestion
 
 loyal_undead:0 "Loyal [GetTrait('being_undead').GetName( GetNullCharacter )|V]"
```